### PR TITLE
Fix etherscan and beaconcha links per network

### PR DESCRIPTION
**Pull request by alexstolr**

### CODE REVIEW

File: src/app/App.tsx [hunk 1]
```diff
@@ -9,21 +9,24 @@ import AppBar from '~app/common/components/AppBar';
 import DeveloperHelper from '~lib/utils/DeveloperHelper';
 import Announcement from '~app/common/components/Announcement';
 import ApplicationStore from '~app/common/stores/Application.store';
+import ErrorBoundary from '../ErrorBoundary';
```
Comment (axelrod-blox) [inline src/app/App.tsx:12]: absolute path

**Review by axelrod-blox — APPROVED** [review]

**Review by IlyaVi — APPROVED** [review]

File: src/app/App.tsx (hunk 1 above)
Comment (alexstolr) [reply src/app/App.tsx]: V

**Review by IlyaVi — APPROVED** [review]

File: src/ErrorBoundary.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                <p>We apologize for the inconvenience. Please try again later.</p>
+              </ErrorMessage>
+            );
+        }
+
+        return this.props.children;
+    }
+}
+
+export default ErrorBoundary;
```
Comment (meshin-dev) [inline src/ErrorBoundary.tsx:51]: Move this file to src/..components

**Review by meshin-dev — APPROVED** [review]

File: src/ErrorBoundary.tsx (hunk 2 above)
Comment (alexstolr) [reply src/ErrorBoundary.tsx:51]: https://github.com/bloxapp/ssv-explorer/pull/99

Comment (alexstolr) [reply src/ErrorBoundary.tsx]: V

**Pull request merged**